locksmith/src/config/config.ts

```diff
@@ -51,6 +51,7 @@ const config = {
       process.env.STORAGE_PUBLIC_HOST || defaultConfig.storage.publicHost,
   },
   recaptchaSecret: process.env.RECAPTCHA_SECRET,
+  logtailSourceToken: process.env.LOGTAIL_SOURCE_TOKEN,
```

Comment:
I am confused about both the config.js and config.ts.
Do we really need both @searchableguy ?

Reply:
yeah because one is used by sequelize-cli and it doesn't support running typescript config file. You don't need to add these to that one.

Reply:
ha! Good point. Ok I think we indeed need to "normalize"... but that's for later!